peps/pep-0733.rst

```diff
@@ -0,0 +1,629 @@
+it was, are difficult for alternative implementations to work with
+[`Issue 64 <https://github.com/capi-workgroup/problems/issues/64>`__].
+Finally, lessons were learned and mistakes in both the design and the
+implementation of the C API were identified.
+
+Evolving the C API is hard due to the combination of backwards
+compatibility constraints and its inherent complexity, both
+technical and social. Different types of users bring different,
+sometimes conflicting, requirements. The tradeoff between stability
+and progress is an ongoing, highly contentious topic of discussion
+when suggestions are made for incremental improvements.
+Several proposals have been put forward for improvement, redesign
+or replacement of the C API, each representing a deep analysis of
+the problems.  At the 2023 Language Summit, three back-to-back
+sessions were devoted to different aspects of the C API. There is
+general agreement that a new design can remedy the problems that
+the C API has accumulated over the last 30 years, while at the same
+time updating it for use cases that it was not originally designed for.
+
+However, there was also a sense at the Language Summit that we are
+trying to discuss solutions without a clear common understanding
+of the problems that we are trying to solve. We decided that
+we need to agree on the current problems with the C API, before
+we are able to evaluate any of the proposed solutions. We
+therefore created the
+`capi-workgroup <https://github.com/capi-workgroup/problems/issues/>`__
+repository on GitHub in order to collect everyone's ideas on that
+question.
+
+Over 60 different issues were created on that repository, each
+describing a problem with the C API. We categorized them and
+identified a number of recurring themes. The sections below
+mostly correspond to these themes, and each contains a combined
+description of the issues raised in that category, along with
+links to the individual issues. In addition, we included a section
+that aims to identify the different stakeholders of the C API,
+and the particular requirements that each of them has.
+
+
+C API Stakeholders
+==================
+
+As mentioned in the introduction, the C API was originally
+created as the internal interface between CPython's
+interpreter and the Python layer. It was later exposed as
+a way for third-party developers to extend and embed Python
+programs. Over the years, new types of stakeholders emerged,
+with different requirements and areas of focus. This section
+describes this complex state of affairs in terms of the
+actions that different stakeholders need to perform through
+the C API.
+
+Universal Actions for All Stakeholders
+--------------------------------------
+
+There are actions which are generic, and required by
+all types of API users:
+
+* Define functions and call them
+* Create classes and instantiate them
+* Create instances of builtin and user-defined types
+  and perform operations on them
```

Comment:
There is some overlap between these two points, or rather, the second seems to include the first.